.github/workflows/release.yml

```
@@ -32,14 +32,22 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Install Rust Toolchain
+      - name: Install Rust Toolchain Stable
         uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
         with:
           toolchain: stable
           default: true
           profile: minimal
           target: wasm32-unknown-unknown
 
+      - name: Install Rust Toolchain Nightly
+        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
+        with:
+          toolchain: nightly-2023-07-13
+          default: false
+          profile: minimal
```

I think `minimal` does not include `rustfmt` this might fix it.
```suggestion
          profile: minimal
          components: rustfmt
```